README.rst

```diff
@@ -114,5 +114,7 @@ This needs to run on a private server because it will have login for smtp and bu
 
 Cronjob::
 
-  00 14 * * 1-5 $HOME/run_autonags_daily.sh &> /tmp/autonag.log
-  15 */1 * * * $HOME/runauto_nag_hourly.sh &> /tmp/autonag-hour.log
+  00 17 * * 2 $HOME/relman-auto-nag/runauto_nag_tuesday.sh &> /tmp/autonag-tuesday.log
```

Comment:
Maybe we should explain very briefly why these times.

Reply:
Done in 89da37c0fe1b98d9c038b6c5c2188a702aca1c68